src/main/groovy/com/calgaryscientific/gradle/VeracodeUploadFile.groovy

```diff
@@ -30,7 +30,7 @@ import groovy.transform.CompileStatic
 
 @CompileStatic
 class VeracodeUploadFile {
-    static void uploadFile(File file, Integer maxTries, Integer waitTime, VeracodeAPI veracodeAPI, File outputFile) {
+    static void uploadFile(File file, Integer maxTries, Integer waitTime, VeracodeAPI veracodeAPI, File outputFile, boolean sandbox) {
```

Comment:
do you need to pass in sandbox? as veracodeAPI would know if  it is sandbox or not?